src/main/java/org/openstreetmap/atlas/geography/atlas/complete/CompleteNode.java

```diff
@@ -31,6 +35,32 @@
     private SortedSet<Long> outEdgeIdentifiers;
     private Set<Long> relationIdentifiers;
 
+    CompleteNode(final long identifier)
+    {
+        this(identifier, null, null, null, null, null);
+    }
+
+    public CompleteNode(final Long identifier, final Location location,
```

Comment:
Is there any chance to reverse that re-ordering (and the other ones)? We order static methods at the top of the class, and here some static methods are being pushed down.

Reply:
Ok, I will move this